flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -1589,7 +1589,7 @@ class ScalarExprLowering {
   /// Helper to lower intrinsic arguments for inquiry intrinsic.
   ExtValue
   lowerIntrinsicArgumentAsInquired(const Fortran::lower::SomeExpr &expr) {
-    if (IsAllocatableOrPointer(expr))
+    if (isAllocatableOrPointer(expr))
```

Comment:
Looks like there should've been a `Fortran::evalaute::` prefix here.

Reply:
I think the intention always was to call `isAllocatableOrPointer`, it was a typo that happened to compile into something reasonable in most cases. It is weird that it was possible to call `IsAllocatableOrPointer` without the namespace prefix here though... There might be a using namepsace leaking somewhere.